git_machete/github.py

```diff
@@ -47,6 +47,7 @@ def _token_from_gh() -> Optional[str]:
     # `gh` can store auth token for public and enterprise domains, specify
     # single domain for lookup.
     # This is *only* github.com until enterprise support is added.
+    # https://github.com/VirtusLab/git-machete/issues/137
```

Comment:
```suggestion
    # TODO (#137): `gh` can store auth token for public and enterprise domains, specify
    # single domain for lookup.
    # This is *only* github.com until enterprise support is added.
```

Rel: see https://virtuslab.github.io/tips/#jetbrains-products/linking-issues-in-jb-ides and https://github.com/VirtusLab/git-machete/blob/master/ci/checks/enforce-issue-number-for-todos.sh